tests/qunit/tests/assets/dev/js/editor/document/component.spec.js

```diff
@@ -59,11 +59,11 @@ QUnit.module( 'Component: document', () => {
 			assert.equal( elementor.documents.getCurrent().history.getItems().length, 3,
 				'Master document still have "3" items in history.' );
 
-			// Put back saved document, to current.
-			elementor.documents.setCurrent( documentBeforeTest );
-			eData.emptyFetch();
-
 			done();
+
```

Comment:
Explain why the `done` is before the real `done`

Reply:
Handled: https://github.com/elementor/elementor/pull/18221